### PR TITLE
Shell Navigation Manager Documentation

### PR DESCRIPTION
**Pull request by lachlanwgordon**
Hi,
Here's a draft of a walk through for Shell Navigation Manager.  It still needs screenshots and a fair bit of editing, but I thought it best to get some feedback on whether you think walkthrough style documentation is the right approach for this, or should it be proper API doc.

I figure while we're in the experimental phase a walkthrough is the most accessible and it follows the form of a few other pages in this project.

I'll hold off on adding any screenshots and finishing the text until we've got the source merged in incase any APIs change.

### CODE REVIEW

File: mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: 'Navigating with Shell Navigation Manager'
```
Comment (Eilon) [inline mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md:2]: BTW you can add your GitHub id here!

```
author: lachlanwgordon
```

Comment (Eilon) [inline mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md:2]:
```suggestion
title: 'Navigating with Shell Navigation Manager - Mobile Blazor Bindings'
```

File: mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Navigating to a page
+Navigating to a page is archived with an instance of the ShellNavigationManager which you can access in your components using the `@inject` directive. Once you have an instance you call NavigateToAsync() with the Uri for the page you want to open.
+
+For example in my AboutPage I want to navigate to the ContactPage.
+
+```
+@inject ShellNavigationManager NavigationManager
+<ContentPage>
+    <Stack
```
Comment (Eilon) [inline mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md:93]: Looks like the sample is cut off here? Was it also meant to show code to call into the NavigationManager and do the navigation?

File: mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md (hunk 1 above)
Comment (lachlanwgordon) [reply mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md]: Cheers, added

File: mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md (hunk 2 above)
Comment (lachlanwgordon) [reply mobile-blazor-bindings/walkthroughs/shell-navigation-manager.md]: Ya, I just gave up half way through typing this sample. I've finished it now.

**Comment by lachlanwgordon** [comment]
I've finished the docs. Can you take a look and let me know if anything needs changing? Or if you just want to edit it yourself go ahead.

> Looking good! Once we're happy with the content, I'm happy to do a full edit pass. I'm also happy to do all the images unless you really, really want to do them 😄 Images are a bit annoying to do with the docs but by all means go for it if you want!

They're not that bad are they? Unless I'm missing something. I've put some in but if there's something awry I won't be offended if you throw them in the bin.

**Comment by Eilon** [comment]
Thank you, I will start reviewing this. With images I just found it annoying to get things framed right, make a high-res and low-res, then caption it, etc. Nothing truly awful, but a bit of a hassle 😄

**Comment by Eilon** [comment]
https://twitter.com/original_ejl/status/1291099557846081538

**Pull request merged**